amd/src/app/block/element.js

```diff
@@ -208,6 +209,70 @@ export default class BlockElement {
         });
     }
 
+    setupBulkEdit() {
```

Comment:
This whole function should just be merged into `setupBulkDelete`. This way you also avoid a lot of duplicate code